server/game/GameActions/PlaceCard.js

```diff
@@ -6,14 +6,9 @@ class PlaceCard extends GameAction {
         super('placeCard');
     }
 
-    canChangeGameState({ player, card, location, bottom = false }) {
-        player = location === 'play area' ? player || card.controller : card.owner;
-
+    canChangeGameState({ card, location }) {
         if(location === 'draw deck') {
-            return (
-                player.drawDeck[0] !== card && !bottom ||
-                player.drawDeck[player.drawDeck.length - 1] !== card && bottom
-            );
+            return true;
```

Comment:
This change implies that moving the top card of your deck to the top of the deck (or bottom to bottom) constitutes a change in board state. Is that correct from a rules perspective?

Reply:
Unsure as to why I had adjusted this a while ago; regardless, reverted in 31fa026dc6b1e7e530b9fea7fef084789d887b57. We may notice later if it was for a reason 😄

Can mark as resolved.

Reply:
Also reverted `PlaceCard.spec.js` in b43c431991dcf7d6ee651be74c8084d00e98b69b (as it was adjusted with the GameAction changes)